lib/std/os/uefi/protocol.zig

```diff
@@ -28,8 +28,8 @@ pub const Ip6Config = @import("protocol/ip6_config.zig").Ip6Config;
 pub const Udp6ServiceBinding = @import("protocol/udp6_service_binding.zig").Udp6ServiceBinding;
 pub const Udp6 = @import("protocol/udp6.zig").Udp6;
 
-pub const HiiDatabase = @import("protocol/hii_database.zig").HiiDatabase;
-pub const HiiPopup = @import("protocol/hii_popup.zig").HiiPopup;
+pub const HiiDatabase = @import("protocol/hii_database.zig").HIIDatabase;
```

Comment:
The formula for constructing title case is to take the snake case ("hii_database") and then capitalize the beginning, every character after the underscores, and then remove the underscores.

So, it should be `HiiDatabase` and not `HIIDatabase`. Even 2 lines above you can see it says `Udp6`, not `UDP6`.

Reply:
that makes sense :+1:. ill fix what vexu pointed out as well, makes more sense for the name to be unified